qemu/tests/timedrift_check_when_crash.py

```diff
@@ -29,9 +30,10 @@ def run(test, params, env):
     sleep_time = float(params.get("sleep_time", 1800))
     deviation = float(params.get("deviation", 5))
     os_type = params["os_type"]
+    ntp_host_cmd = params.get("ntp_host_cmd", ntp_cmd)
 
     error_context.context("sync host time with ntp server", logging.info)
-    process.system(ntp_cmd, shell=True)
+    process.system(ntp_host_cmd, shell=True)
```

Comment:
Seems this host execution should work for both windows and linux, so the `ntp_host_cmd` should be defined out of `Windows` variants to apply for all os.

Reply:
@YanhuiMa ntp_cmd for windows guest.  ntp_host_cmd on host.  so should move ntp_host_cmd　out of windows variants.  I agree with Qianqian.

For others.  looks good to me.

Reply:
@vivianQizhu @xiangchunfu Because for linux guest, ntp cmds for guest and host are same. No need to define ntp_host_cmd, I just use ntp_cmd "ntp_host_cmd = params.get("ntp_host_cmd", ntp_cmd)".